Makefile

```diff
@@ -241,20 +285,33 @@ deployment-manifest: build-installer ## Generate manifest to deploy operator. De
 BUNDLE_IMGS ?= $(BUNDLE_IMG)
 
 # The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
-CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:$(VERSION)
+CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:$(IMAGE_TAG_VERSION)
 
 # Set CATALOG_BASE_IMG to an existing catalog image tag to add $BUNDLE_IMGS to that image.
 ifneq ($(origin CATALOG_BASE_IMG), undefined)
 FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
 endif
 
+.PHONY: bundles
+bundles: ## Generate bundle manifests and metadata, then validate generated files for all available profiles.
+	@for profile in $(PROFILES); do \
+  		$(MAKE) bundle PROFILE=$$profile; \
+  	done
+
 .PHONY: bundle
 bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metadata, then validate generated files.
 	#$(OPERATOR_SDK) generate kustomize manifests -q
 	cd config/profile/$(PROFILE) && $(KUSTOMIZE) edit set image controller=$(IMG)
+	@rm -f ./bundle/$(PROFILE)/manifests/$(PROFILE)-operator.clusterserviceversion.yaml || true
 	$(KUSTOMIZE) build config/manifests/$(PROFILE) | $(OPERATOR_SDK) generate bundle --kustomize-dir config/manifests/$(PROFILE) $(BUNDLE_GEN_FLAGS)
 	$(OPERATOR_SDK) bundle validate ./bundle/$(PROFILE)
-	mv -f bundle.Dockerfile ./bundle/$(PROFILE)/bundle.Dockerfile
+	@mv -f bundle.Dockerfile ./bundle/$(PROFILE)/bundle.Dockerfile
+	@sed -i 's/-controller-manager/-operator/g' ./bundle/$(PROFILE)/manifests/*.yaml
+	@sed -i 's/: controller-manager/: $(PROFILE_SHORT)-operator/g' ./bundle/$(PROFILE)/manifests/*.yaml
+	@sed -i 's/backstage-operator/$(PROFILE_SHORT)-operator/g' ./bundle/$(PROFILE)/manifests/*.yaml
+	@mv -f ./bundle/$(PROFILE)/manifests/backstage-operator.clusterserviceversion.yaml ./bundle/$(PROFILE)/manifests/$(PROFILE)-operator.clusterserviceversion.yaml
```

Comment:
Is it really necessary to concatenate the file name with $(PROFILE), like: $(PROFILE)-operator.clusterserviceversion.yaml , maybe better leave it predefined as operator.clusterserviceversion.yaml (for example), same way as for other files?

Reply:
Sure, we can do that. No strong opinion. Let me look into that.

Reply:
I've removed this `mv` call. But note that the resulting file name depends on the project name, so it will be `backstage-operator.clusterserviceversion.yaml`. I couldn't find a way to customize this name other than changing the project name.